_includes/banner.html

```diff
@@ -10,6 +10,10 @@
         <span>&nbsp;&nbsp;</span>
         <a href="/documentation/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-book"></i> Documentation</button></a>
         <span>&nbsp;&nbsp;</span>
+        <a href="/resources/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-external-link-square"></i> Resources</button></a>
+        <span>&nbsp;&nbsp;</span>
+        <a href="/resources/"><button type="button" class="btn btn-lg btn-primary"><i class="fa fa-rss-square"></i> Ehcache Blog</button></a>
```

Comment:
I think there are too many top-level menu items now.

What about putting resources under "docs"  and having a "blogs"  with "ehcache blog" and "terracotta blog" under it?